docs/timeseries.md

```diff
@@ -2,6 +2,78 @@
 
 Retrieve time series data from SWOT observations for reaches and nodes.
 
+## Request Headers
+
+### Accept
+
+Accept headers: `application/json`, `text/csv`, `application/geo+json`
+
+Possible header and request parameter combinations:
+
+- If the Accept header is `text/csv` or `applicatin/geo+json`, the raw CSV or GeoJSON response is returned.
+- If the Accept header is `application/json` with an output field of `geojson`, the entire JSON object with metadata including GeoJSON response is returned.
+- If the Accept header is `application/json` with an output field of `csv`, the entire JSON object with metadata including CSV response is returned.
+- If the Accept header is `application/json` without an output field, the entire JSON object with metadata including GeoJSON response is returned.
+- If the Accept header is none of the accepted types then a 415 Unsupported is returned.
+
+Example GeoJSON request and response:
```

Comment:
Your descriptions of the code blocks here is reversed. This is the CSV example

Reply:
@frankinspace - I missed this last week but I think it is fixed now. Do you think the PR is ready to be merged? Merge conflicts should be resolved as well.